packages/cli/src/oclif/commands/invoke.js

```diff
@@ -0,0 +1,685 @@
+const fs = require('node:fs');
+
+const _ = require('lodash');
+const { flags } = require('@oclif/command');
+const debug = require('debug')('zapier:invoke');
+const dotenv = require('dotenv');
+
+// Datetime related imports
+const chrono = require('chrono-node');
+const { DateTime, IANAZone } = require('luxon');
+
+const BaseCommand = require('../ZapierBaseCommand');
+const { buildFlags } = require('../buildFlags');
+const { localAppCommand } = require('../../utils/local');
+const { startSpinner, endSpinner } = require('../../utils/display');
+
+const ACTION_TYPE_PLURALS = {
+  trigger: 'triggers',
+  search: 'searches',
+  create: 'creates',
+};
+
+const ACTION_TYPES = ['auth', ...Object.keys(ACTION_TYPE_PLURALS)];
+
+const AUTH_FIELD_ENV_PREFIX = 'authData_';
+
+const NUMBER_CHARSET = '0123456789.-,';
+
+const FALSE_STRINGS = new Set([
+  'noo',
+  'no',
+  'n',
+  'false',
+  'nope',
+  'f',
+  'never',
+  'no thanks',
+  'no thank you',
+  'nul',
+  '0',
+  'none',
+  'nil',
+  'nill',
+  'null',
+]);
+
+const TRUE_STRINGS = new Set([['yes', 'yeah', 'y', 'true', 't', '1']]);
+
+const loadAuthDataFromEnv = () => {
+  return Object.entries(process.env)
+    .filter(([k, v]) => k.startsWith(AUTH_FIELD_ENV_PREFIX))
+    .reduce((authData, [k, v]) => {
+      const fieldKey = k.substr(AUTH_FIELD_ENV_PREFIX.length);
+      authData[fieldKey] = v;
+      return authData;
+    }, {});
+};
+
+const readStream = async (stream) => {
+  const chunks = [];
+  for await (const chunk of stream) {
+    chunks.push(chunk);
+  }
+  return chunks.join('');
+};
+
+const getMissingRequiredInputFields = (inputData, inputFields) => {
+  return inputFields.filter(
+    (f) => f.required && !f.default && !inputData[f.key]
+  );
+};
+
+const getMissingOptionalInputFields = (inputData, inputFields) => {
+  return inputFields.filter((f) => !f.required && !inputData[f.key]);
+};
+
+const parseDecimal = (s) => {
+  const chars = [];
+  for (const c of s) {
+    if (NUMBER_CHARSET.includes(c)) {
+      chars.push(c);
+    }
+  }
+  const cleaned = chars.join('').replace(/[.,-]$/, '');
+  return parseFloat(cleaned);
+};
+
+const parseInteger = (s) => {
+  const n = parseInt(s);
+  if (!isNaN(n)) {
+    return n;
+  }
+  return Math.floor(parseDecimal(s));
+};
+
+const parseBoolean = (s) => {
+  s = s.toLowerCase();
+  if (TRUE_STRINGS.has(s)) {
+    return true;
+  }
+  if (FALSE_STRINGS.has(s)) {
+    return false;
+  }
+  return Boolean(s);
+};
```

Comment:
I found this a bit unintuitive - even though it looks like it should help.
For example, 
```
$ zl invoke create recipe --inputData '{"name": "eggs", "style": "no" }'
✔ Invoking creates.recipe.operation.perform
{
  "name": "eggs",
  "music": "pop",
  "fish_type": "qui est esse",
  "style": false,
  "id": "RXU237w"
}

$ zl invoke create recipe --inputData '{"name": "eggs", "style": "noo" }'
✔ Invoking creates.recipe.operation.perform
{
  "name": "eggs",
  "music": "pop",
  "fish_type": "qui est esse",
  "style": false,
  "id": "sqAM~d4"
}

$ zl invoke create recipe --inputData '{"name": "eggs", "style": "nooo" }'
✔ Invoking creates.recipe.operation.perform
{
  "name": "eggs",
  "music": "pop",
  "fish_type": "qui est esse",
  "style": true,
  "id": "4dDws4N"
}
```


Does it make more sense to have strict true/false validation here?

Reply:
I wrote it this way to match this [function](https://gitlab.com/zapier/zapier/-/blob/c78527c19d859daa8c19c327d10bf4972b2478d4/web/backend/zapier/useful/template_tools.py#L1079) in production.